lib/types/space.types.ts

```diff
@@ -32,86 +32,291 @@ type Snapshot<T> = {
+  /**
+   * @deprecated since version 4.0.0
+   * Consider using `cma.asset.delete` instead
+   * See https://www.contentful.com/developers/docs/extensibility/app-framework/sdk/#using-the-contentful-management-library for more details
+   */
   deleteAsset: (data: Asset) => Promise<void>
+  /**
+   * @deprecated since version 4.0.0
+   * Consider using `cma.asset.publish` instead
+   * See https://www.contentful.com/developers/docs/extensibility/app-framework/sdk/#using-the-contentful-management-library for more details
+   */
   publishAsset: (data: Asset) => Promise<Asset>
+  /**
+   * @deprecated since version 4.0.0
+   * Consider using `cma.asset.unpublish` instead
+   * See https://www.contentful.com/developers/docs/extensibility/app-framework/sdk/#using-the-contentful-management-library for more details
+   */
   unpublishAsset: (data: Asset) => Promise<Asset>
+  /**
+   * @deprecated since version 4.0.0
+   * Consider using `cma.asset.archive` instead
+   * See https://www.contentful.com/developers/docs/extensibility/app-framework/sdk/#using-the-contentful-management-library for more details
+   */
   archiveAsset: (data: Asset) => Promise<Asset>
+  /**
+   * @deprecated since version 4.0.0
+   * Consider using `cma.asset.processForLocale` instead
+   * See https://www.contentful.com/developers/docs/extensibility/app-framework/sdk/#using-the-contentful-management-library for more details
+   */
   processAsset: (data: Asset, locale: string) => Promise<Asset>
+  /**
+   * @deprecated since version 4.0.0
+   * Consider using `cma.asset.unarchive` instead
+   * See https://www.contentful.com/developers/docs/extensibility/app-framework/sdk/#using-the-contentful-management-library for more details
+   */
   unarchiveAsset: (data: Asset) => Promise<Asset>
+  /**
+   * @deprecated since version 4.0.0
+   * Consider using `cma.asset.getMany` instead
+   * See https://www.contentful.com/developers/docs/extensibility/app-framework/sdk/#using-the-contentful-management-library for more details
+   */
   getPublishedAssets: <Query extends SearchQuery = SearchQuery>(
     query?: Query
   ) => Promise<CollectionResponse<Asset>>
+  /**
+   * @deprecated since version 4.0.0
+   * Consider using `cma.upload.create` instead
+   * See https://www.contentful.com/developers/docs/extensibility/app-framework/sdk/#using-the-contentful-management-library for more details
+   */
   createUpload: (base64data: string) => Promise<void>
+  /**
+   * @deprecated since version 4.0.0
+   * Consider using TODO - what method?
+   * See https://www.contentful.com/developers/docs/extensibility/app-framework/sdk/#using-the-contentful-management-library for more details
+   */
   waitUntilAssetProcessed: (assetId: string, locale: string) => Promise<Asset>
 
-  /** Returns all users who belong to the space. */
+  /**
+   * @deprecated since version 4.0.0
+   * Consider using `cma.user.getManyForSpace` or `cma.user.getManyForOrganization` instead
```

Comment:
can you do `getManyForOrg`? considering the space env scope of apps

Reply:
no! good point. :)